numpy/core/include/numpy/ndarraytypes.h

```diff
@@ -1282,6 +1282,14 @@ typedef struct {
 #define PyArray_MultiIter_NOTDONE(multi)                \
         (_PyMIT(multi)->index < _PyMIT(multi)->size)
 
+#define PyArray_MultiIter_SIZE(multi) (_PyMIT(multi)->size)
+#define PyArray_MultiIter_NDIM(multi) (_PyMIT(multi)->nd)
+#define PyArray_MultiIter_INDEX(multi) (_PyMIT(multi)->index)
+#define PyArray_MultiIter_NUMITER(multi) (_PyMIT(multi)->numiter)
+#define PyArray_MultiIter_DIMS(multi) (_PyMIT(multi)->dimensions)
+#define PyArray_MultiIter_ITERS(multi) (_PyMIT(multi)->iters)
```

Comment:
At some point inline functions were preferred.  I don't actually know there is a good reason for that although I like functions.  @mattip do you know a reason?